test/e2e/app-dir/app-static/app/dynamic-param-edge/[slug]/page.tsx

```diff
@@ -5,9 +5,7 @@ export default async function Hello({ params }) {
 export function generateStaticParams() {
```

Comment:
We don't call `generateStaticParams` in edge runtime so might be worth extra validation if `runtime = 'edge'` and `generateStaticParams` is specified.